### ISSUE TITLE
[Feature] Criar sistema de reputação

### ISSUE THREAD

**Issue by tibawatanabe** [indigotech/bot-devices#18]
# Contexto

Muitas vezes os devices são devolvidos com proxy, air-plane mode ativo, etc.
# Objetivo

Criar um sistema onde a pessoa que pega um device possa avaliar o "estado" do mesmo, fazendo com que o owner anterior ganhe ou perca reputação.
# Entraves

Há pessoas que pegam o device e não marcam no bot, isso poderia causar avaliações de pessoas erradas.

**Comment by askmon** [comment]
Será que adicionar um lembrete quando a pessoa faz o device-back nao ajudaria tbm?

**Comment by tibawatanabe** [comment]
Essa é uma possibilidade. Mas será q as pessoas irão ler esse lembrete?
Outra possibilidade é fazer uma série de perguntas ao user no `back`. Mas isso pode deixar o processo com um overhead chato...

**Comment by melloflavio** [comment]
Concordo um pouco com o Tiba. Imagino alguém que pegou diversos devices para testar versões distintas do sistema dando um `device-back all` e passando 15min respondendo formulário...
